test/onnx_import.cpp

```diff
@@ -74,6 +75,24 @@ TEST(onnx, model_addmul_abc)
     EXPECT_TRUE(test::all_close_f(expected_output, result_vectors.front()));
 }
 
+TEST(onnx, model_argmin_no_keepdims)
+{
+    Shape input_shape{2, 2};
+    Shape result_shape{2};
+
+    auto function = onnx_import::import_onnx_function(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/argmin_no_keepdims.onnx"));
+
+    auto backend = runtime::Backend::create("INTERPRETER");
+
+    auto input_tensor = backend->create_tensor(element::f32, input_shape);
+    copy_data(input_tensor, std::vector<float>{2, 1, 3, 10});
+    auto result_tensor = backend->create_tensor(element::i64, result_shape);
+
+    backend->call_with_validate(function, {result_tensor}, {input_tensor});
```

Comment:
Why you're not using `execute` function? It's capable to handle different types for input and output tensors. Look here https://github.com/NervanaSystems/ngraph/blob/01205771e8b12024a013dc1f87fe4305327dab46/test/onnx_import.cpp#L972.